packages/components/status/src/status.css

```diff
@@ -0,0 +1,27 @@
+/* General........................................................ */
+
+:host {
+}
+
+.status {
+	padding: 3px 10px 3px 10px;
+	font-size: var(--ts-fontsize-small);
+	text-transform: uppercase;
+	font-weight: var(--ts-fontweight-semibold);
+	border-radius: 3px;
+}
+
+.status-ok {
+	color: #287d06;
+	background-color: #e2f5d8;
+}
+
+.status-error {
+	color: #d15b00;
+	background-color: #ffecd6;
+}
+
+.status-note {
+	color: #0065a3;
+	background-color: #d6f2ff;
```

Comment:
Please, use colours from the `packages/core/src/vars.css`, there are colours approved by our designers.

Reply:
ok